drizzlepac/make_custom_mosaic.py

```diff
@@ -0,0 +1,428 @@
+        Dictionary containing projection cell information
+
+    Returns
+    -------
+    poller_filename : str
+        Name of the newly created poller_filename
+    """
+    # Locate bottom-left most skycell
+    pcell_filename = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'pars', 'allsky_cells.fits')
+    sc_nxy = fits.getval(pcell_filename, "SC_NXY",
+                         ext=0)  # Projection cell side length (in skycells) in X or Y
+    closest_skycell = ""
+    min_dist = calc_skycell_dist(sc_nxy, sc_nxy, 1, 1) + 2.0
+    for sc in proj_cell_dict.keys():
+        x_index = proj_cell_dict[sc].x_index
+        y_index = proj_cell_dict[sc].y_index
+
+        dist = calc_skycell_dist(x_index, y_index, 1, 1)
+        if dist < min_dist:
+            min_dist = dist
+            closest_skycell = sc
+
+    # Create poller file
+    skycell_name = closest_skycell.replace("skycell-", "")
+    poller_filename = "temp_{}_mvm.out".format(skycell_name)
+    rootname_list = []
+    for imgname in img_list:
+        rootname_list.append(imgname.split("_")[0]+"\n")
+    tf = tempfile.NamedTemporaryFile(mode='w+t', dir=os.getcwd())
+    with open(tf.name, 'w') as f:
+        f.writelines(rootname_list)
+        f.close()
+
+        make_poller_files.generate_poller_file(tf.name, poller_file_type="mvm",
+                                               output_poller_filename=poller_filename,
+                                               skycell_name=skycell_name)
+    return poller_filename
+
+# ------------------------------------------------------------------------------------------------------------
+
+
+def compute_mosaic_limits(proj_cell_dict):
+    """Compute min and max limits of the rectangle that encloses the mosaic observations
+
+    Parameters
+    ----------
+    proj_cell_dict : dictionary
+        Dictionary containing projection cell information
+
+    Returns
+    -------
+    mosaic_limits : list
+        4-element list containing the mosaic bounding rectangle X min and max and Y min and max values
+    """
+    # set up storage arrays
+    array_size = len(proj_cell_dict.keys()) * 4
+    ra_values = np.empty(array_size)
+    dec_values = np.empty(array_size)
+    x_values = np.empty(array_size)
+    y_values = np.empty(array_size)
+
+    # Use wcs.calc_footprint() to get skycell corners, convert RA, Dec to X, Y in projection cell frame of reference
+    i = 0
+    for sc_name in proj_cell_dict.keys():
+        for ra_dec in proj_cell_dict[sc_name].wcs.calc_footprint().tolist():
+            x_y = proj_cell_dict[sc_name].projection_cell.wcs.all_world2pix([ra_dec], 0).tolist()[0]
+            ra_values[i] = ra_dec[0]
+            dec_values[i] = ra_dec[1]
+            x_values[i] = x_y[0]
+            y_values[i] = x_y[1]
+            i += 1
+
+    # Return mosaic bounding rectangle X min and max and Y min and max values
+    mosaic_limits = [x_values.min(), x_values.max(), y_values.min(), y_values.max()]
```

Comment:
Logic needs to be added to insure that these computed limits do not extend beyond the limits of the projection cell; namely, only go from 0 to NX and 0 to NY.

Reply:
Issue resolved by commit [a977171](https://github.com/spacetelescope/drizzlepac/pull/1097/commits/a977171d520d1f46acf89f997e76998ae699e1b3)